frame/contracts/src/wasm/prepare.rs

```diff
@@ -439,10 +394,6 @@ where
 		contract_module.ensure_parameter_limit(schedule.limits.parameters)?;
 		contract_module.ensure_br_table_size_limit(schedule.limits.br_table_size)?;
 
-		if matches!(determinism, Determinism::Deterministic) {
-			contract_module.ensure_no_floating_types()?;
-		}
```

Comment:
We no longer need this because the newest `wasmparser` rejects floating types for us if `floats: false`. Before it only rejected float instructions.

Reply:
just curious, where do you define this setting?

Reply:
Just a bit above. Argument to `Validator::new_with_features`.